src/zeit/retresco/connection.py

```diff
@@ -171,12 +182,19 @@ def _build_topic_xml():
     for row in tms.get_all_topicpages():
         # XXX What other attributes might be interesting to use in a
         # dynamicfolder template?
-        root.append(E.topic(
-            row['title'],
-            id=zeit.cms.interfaces.normalize_filename(row['name'])))
+        root.append(E.topic(row['title'], id=row['id']))
     return root
 
 
+class Topicpages(object):
+
+    zope.interface.implements(zeit.cms.tagging.interfaces.ITopicpages)
+
+    def get_topics(self, start=0, rows=25):
+        tms = zope.component.getUtility(zeit.retresco.interfaces.ITMS)
+        return tms.get_topicpages(start, rows)
+
+
```

Comment:
Bei `zeit.intrafind` wurden die `Topicpages` noch in der `configure.zcml` registriert (sogar als Utility, von der ich im Moment noch keine Verwendung sehe.) Ist das hier nicht  nötig?